js/web/test/e2e/browser-test-wasm-Image-tensor-Image.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+'use strict';
+
+it('Browser E2E testing - Tensor <--> Image E2E test', async function () {
+  // Image URL to tensor API
+  const inputTensor = await ort.Tensor.fromImage('https://onnxruntime.ai/images/ONNX-Icon.png');
```

Comment:
not sure if this is a good idea - the test is now depending on internet availability.

Reply:
Do you suggest we use an asset in the test ? after all it's a URL test API so either we add an asset to the test or we use the internet.

Reply:
yes, use an asset and serve it in the E2E is a better idea.

Reply:
there are also plenty of png's under onnxruntime/docs

Reply:
I changed the test to generate white noise image so now we don't rely on internet connection (only in the case of URL -->tensor)